prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/FhirBundleUtils.kt

```diff
@@ -64,7 +64,7 @@ object FhirBundleUtils : Logging {
         } else if (StringCompatibleType.values().any { it.typeAsString == sourceType }) {
             getValue(targetType, value, logger, sourceType, true) ?: value
         } else {
-            logger.error("Conversion between $sourceType and $targetType not yet implemented.")
+            logger.debug("Conversion between $sourceType and $targetType not yet implemented.")
```

Comment:
Not super sure why these were errors; they're not alerted on, don't cause any issues, but are logged _all_ the time; I opted to turn them down to debug so we had less noise in the logs